PermissionsScraper/PermissionsScraper/Common/StringExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿// ------------------------------------------------------------------------------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------------------------------------------------------------------------------
+
+namespace PermissionsScraper.Common
+{
+    public static class StringExtensions
+    {
+        /// <summary>
+        /// Change the input string's line breaks
+        /// </summary>
+        /// <param name="rawString">The raw input string</param>
+        /// <param name="newLine">The new line break.</param>
+        /// <returns>The changed string.</returns>
+        public static string ChangeLineBreaks(this string rawString, string newLine = "\n")
```

Comment:
maybe https://docs.microsoft.com/en-us/dotnet/api/system.environment.newline?view=net-5.0  ?

Reply:
Are you suggesting I use `Environment.NewLine` instead of `\n` as the default value?
The default value needs to be a compile-time constant.